src/xenia/ui/imgui_drawer.cc

```diff
@@ -218,6 +196,130 @@ std::optional<ImGuiKey> ImGuiDrawer::VirtualKeyToImGuiKey(VirtualKey vkey) {
   }
 }
 
+static const ImWchar font_glyph_ranges[] = {
+    0x0020, 0x00FF,  // Basic Latin + Latin Supplement
+    0x0370, 0x03FF,  // Greek
+    0x0400, 0x044F,  // Cyrillic
+    0x2000, 0x206F,  // General Punctuation
+    0,
+};
+
+const std::filesystem::path ImGuiDrawer::GetWindowsFont(std::string font_name) {
+  std::filesystem::path font_path = "";
+
+#if XE_PLATFORM_WIN32
+  PWSTR fonts_dir;
+  HRESULT result = SHGetKnownFolderPath(FOLDERID_Fonts, 0, NULL, &fonts_dir);
+  if (FAILED(result)) {
+    CoTaskMemFree(static_cast<void*>(fonts_dir));
```

Comment:
`fonts_dir` would likely (though I'm not entirely sure about this specific case, of course) only be initialized if the function has succeeded, this freeing may cause a crash.

Reply:
https://learn.microsoft.com/en-us/windows/win32/api/shlobj_core/nf-shlobj_core-shgetknownfolderpath

``The calling process is responsible for freeing this resource once it is no longer needed by calling [CoTaskMemFree](https://learn.microsoft.com/en-us/windows/desktop/api/combaseapi/nf-combaseapi-cotaskmemfree), whether SHGetKnownFolderPath succeeds or not``